modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/util/VelocityMigrationUtil.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check.util;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringBundler;
+import com.liferay.source.formatter.util.FileUtil;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * @author Nícolas Moura
+ */
+public class VelocityMigrationUtil {
+
+	public static String getFTLContentMigrated(
+			String content, String velocityFileName)
+		throws IOException {
+
+		File file = _getMigratedFTLFile(velocityFileName);
```

Comment:
@nicolas-sss, how about we use the term `FreeMarker` instead of just `FTL`?

We're working on a Velocity to FreeMarker converter and we're naming one as Velocity and the other one as FTL, which seems to be a little inconsistent.

What do you think?